std_msgs/Header header

float32 x
float32 y
float32 radius
int16 frame_width
int16 frame_height

string color
